### PR TITLE
Try both python and python3

### PR DESCRIPTION
**Pull request by Androbin**
Used `configure` script for reference
Reopens mistakenly closed #10525

### CODE REVIEW

File: tools/tf_env_collect.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ set -u  # Check for undefined variables
 echo "Collecting system information..."
 
 OUTPUT_FILE=tf_env.txt
+python_bin_path=$(which python || which python3 || true)
```
Comment (martinwicke) [inline tools/tf_env_collect.sh]: If we don't find python or python3, is it reasonable to not error out?

Comment (Androbin) [inline tools/tf_env_collect.sh]: Good point, had just taken from [`configure#L76`](https://github.com/tensorflow/tensorflow/blob/master/configure#L76)
So, should we display an appropriate error message?

Comment (martinwicke) [inline tools/tf_env_collect.sh]: It's probably a super-rare case, but that would be nicest.

**Review by martinwicke — APPROVED** [review]

**Comment by martinwicke** [comment]
skipping tests since this is untested.

**Pull request merged**